### ISSUE TITLE
feature: include enum values in the error message of IsEnum() decorator

### ISSUE THREAD

**Issue by rafaelbeckel** [typestack/class-validator#837]
### Description
`IsEnum(obj)` will produce the same error message, regardless of the expected enum values, which is useless for any practical user-facing application. Returning the message **"{obj} must be a valid enum value"** does not say anything about the expected input and is not actionable (unless there are some supporting docs somewhere else).

### Proposed solution
It should behave like `IsIn(Object.values(obj))`, which returns the message **"{obj} must be one of the following values: cupcakes,lollipops,candies,whatever"**. This is much nicer and actionable. The end-user cares about the expected values only, not about how they are represented internally.

**Comment by NoNameProvided** [comment]
Hi! Yeah that is a good idea, although a simple `Object.values` call won't work because an enum is stored like this: 

```js
const MyEnum = {
  '1': "FirstValue",
  '2': "SecondValue",
  "FirstValue": 1,
  "SecondValue": 2
}
```

(Except if it's a string enum, then it's stored differently.)

This is a good first issue btw if you want to take a look at it.

**Comment by hood** [comment]
Feature implemented in #887. Waiting for review and approval.

**Comment by rpihlak** [comment]
> Feature implemented in #887. Waiting for review and approval.

When is the pull request going to be merged?

**Comment by hood** [comment]
> > Feature implemented in #887. Waiting for review and approval.
> 
> When is the pull request going to be merged?

It's been quite a while, right? :thinking:

**Comment by NoNameProvided** [comment]
This feature/fix has been released in `0.14.0`.
